core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1942,13 +1942,14 @@ private final class SubchannelImpl extends AbstractSubchannel {
     ScheduledHandle delayedShutdownTask;
 
     SubchannelImpl(CreateSubchannelArgs args, LbHelperImpl helper) {
+      checkNotNull(args, "args");
       addressGroups = args.getAddresses();
```

Comment:
Above 2 lines can be combined as

```
addressGroups = checkNotNull(args, "args").getAddresses();
```

??

Reply:
I agree, but it would also be brittle as lines move. I'm fine with accepting as-is.